documentation/en/source/pages/templates-spec/stack-bundles.rst

```diff
@@ -205,10 +205,20 @@ To get the list of categories available, run command:
 
 	$ hammr bundle categories
 
-Adding oses
-~~~~~~~~~~~
+Adding restriction rule
+~~~~~~~~~~~~~~~~~~~~~~~
 
-The following example show how you can provide oses for which the bundle is designed.
+The following example show how you can provide restrictions.
```

Comment:
The following example shows how you can set rules to restriction the bundle to specific distributions and/or target formats. For a list of supported distribution and image formats refer to :ref: